src/components-v2/YieldBearingVaults/YieldBearingRewards.tsx

```diff
@@ -62,15 +69,24 @@ const YieldBearingRewards = observer(({ open, onModalClose }: YieldBearingReward
       aria-describedby="alert-dialog-description"
     >
       <DialogTitle id="alert-dialog-title" className={classes.title}>
-        <Typography variant="h6" color="primary" className={classes.titleText}>
-          <img src="assets/icons/yield-bearing-rewards.svg" alt="Yield-Bearing Rewards" /> Yield-Bearing Rewards
-        </Typography>
+        <Box className={classes.titleText}>
+          <img src="assets/icons/yield-bearing-rewards.svg" alt="Yield-Bearing Rewards" />
```

Comment:
this needs an `alt` field

Reply:
good catch

Reply:
wait did you mean to link something else? l73 has an alt

Reply:
no i am blind